File: trail1/완전수 판별기/perfect-number-discriminator.cs
using System;

public class Codetree
{  
    public static void Main()
    {   
        int N = int.Parse(Console.ReadLine());
        int ssum = 0;
        for (int i=1; i<N; i++) {
            if (N % i == 0) {ssum += i;}
        }
        char val = (ssum == N) ? 'P' : 'N';
        Console.WriteLine(val); 
    }
}

File: trail1/완전수 판별기/perfect-number-discriminator.c
#include <stdio.h>

int main() {
    int N;
    int ssum = 0;
    scanf("%d", &N);
    for (int i=1; i<N; i++) {
        if (N % i == 0) {ssum += i;}
    }
    char val = ssum == N ? 'P' : 'N';
    printf("%c", val);
    return 0;
}